mllib/src/main/scala/org/apache/spark/ml/image/ImageSchema.scala

```diff
@@ -133,7 +133,13 @@ object ImageSchema {
    */
   private[spark] def decode(origin: String, bytes: Array[Byte]): Option[Row] = {
 
-    val img = ImageIO.read(new ByteArrayInputStream(bytes))
+    val img = try {
+      ImageIO.read(new ByteArrayInputStream(bytes))
+    } catch {
+      // Catch runtime exception because `ImageIO` may throw unexcepted `RuntimeException`.
+      // But do not catch the declared `IOException` (regarded as FileSystem failure)
+      case _: RuntimeException => null
```

Comment:
log the error message?  may help with debugging

Reply:
I am afraid it will log too many things , suppose loading millions of images. (And when loading failed with `imageIO.read` normally return "null" we also do not log error.